internal/acceptance/workflows/testdata/tiles/v2/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/crhntr/hello-tile
 
-go 1.20
```

Comment:
As far as I know, you don't need to bump this. When run with a newer version, Go should just run in a sort of compatibility mode.

Reply:
Thanks Chris.  Will you be merging this?